packages/cli/README.md

```diff
@@ -0,0 +1,92 @@
+# @mermaidchart/cli
+
+CLI for interacting with https://MermaidChart.com, the platform that makes collaborating with Mermaid diagrams easy!
+
+## Installation
+
+`@mermaidcart/cli` is tested to work with Node.JS v18.18.0 or later.
+
+We recommend installation using [npx](https://docs.npmjs.com/cli/v10/commands/npx)
+to automatically download, cache, and run `@mermaidcart/cli`, as it comes with
+most Node.JS installations.
+
+```bash
+npx @mermaidchart/cli --help
+```
+
+## Usage
+
+```bash
+npx @mermaidchart/cli <command>
+```
+
+Use `--help` to see options!
+
+```bash
+npx @mermaidchart/cli --help
+```
+
+`@mermaidchart/cli` allows you to easily sync local diagrams with your diagrams
+on https://mermaidchart.com.
+
+### `login`
+
+Firstly, go to https://www.mermaidchart.com/app/user/settings and generate an
+API key, which you can then setup by running:
+
+```bash
+npx @mermaidchart/cli login
+```
```

Comment:
Can we use OAuth before releasing?

Reply:
IMO, since it's not a mandatory feature that's block the v0.1.0 release, I think it can wait for a future minor release (e.g. v0.1.1)!

But yep, we should definitely add it to make it easier for users to sign-in!